### PR TITLE
[#715] `VaCheckbox` visual updates

### PR DESCRIPTION
**Pull request by aluarius**
Related: #715

## Description
Minor `VaCheckbox` visual updates.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

**Review by rustem-nasyrov — APPROVED** [review]

**Comment by m0ksem** [comment]
image

We will destroy users layout which rely on this padding on right!
---

image

Is it planned to remove color from error messages?
---
image
Small ugly space

**Comment by m0ksem** [comment]
Revert display inline-block.

**Review by rustem-nasyrov — APPROVED** [review]

File: packages/ui/src/components/va-checkbox/VaCheckbox.demo.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <VbDemo>
+  <VbDemo class="va-input-demo">
```
Comment (m0ksem) [inline packages/ui/src/components/va-checkbox/VaCheckbox.demo.vue:2]:
```suggestion
  <VbDemo class="d-flex direction--column">
```

But, do we actually need it if checkbox is block?

**Review by m0ksem — APPROVED** [review]

File: packages/ui/src/components/va-checkbox/VaCheckbox.demo.vue (hunk 1 above)
Comment (aluarius) [reply packages/ui/src/components/va-checkbox/VaCheckbox.demo.vue:2]: I've decided not delete it - we will need it after we make checkbox inline.

**Pull request merged**